### PR TITLE
added few of the welsh content missing and fixed bug

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/respondent/safety_concerns/domestic_abuse_risk_no/content.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ const cy: typeof en = {
   line14: 'If you are a man and have concerns for your safety',
   line15: 'Llinell Gymorth Genedlaethol Trais Domestig',
   line16: 'If you want to change your answer, go back to the previous screen.',
-  continue: 'Continue',
+  continue: 'Parhau',
   goBack: 'Go back',
```
Comment (vivek-sekhar) [inline src/main/steps/respondent/safety_concerns/domestic_abuse_risk_no/content.ts:49]: translation missing check full file

File: src/main/steps/respondent/task-list/utils.ts [hunk 2]
```diff
@@ -275,6 +275,40 @@ export const getRespondentPartyDetailsCa = (userCase: Partial<CaseWithId>, userI
   return undefined;
 };
 
+export const getResponseStatus = (userCase: Partial<CaseWithId> | undefined, userId: string): SectionStatus => {
+  const respondent1 = userCase?.respondents?.find(respondent => {
```
Comment (vivek-sekhar) [inline src/main/steps/respondent/task-list/utils.ts:279]: please change the const to respondent and inner argument of find method to _respondent

Comment (SasankAchyuta9) [reply src/main/steps/respondent/task-list/utils.ts:279]: Yes this is taken care.

File: src/main/steps/complete-your-application-paper-form/content.ts [hunk 3]
```diff
@@ -12,12 +12,12 @@ const en = () => ({
...
 
 const cy = () => ({
-  section: 'What to do next -welsh',
-  title: 'Complete your application using a paper form -welsh',
-  subtitle: "you can do this using a child arrangements application form (also known as 'Form C100'). -welsh",
+  section: 'Beth i’w wneud nesaf',
+  title: 'Cwblhau eich cais gan ddefnyddio ffurflen bapur',
+  subtitle: 'Gallwch wneud hyn gan ddefnyddio ffurflen gais trefniadau plant (a elwir hefyd yn ‘Ffurflen C100’).',
   giveDetails:
     "See <a href='https://www.gov.uk/government/publications/form-c100-application-under-the-children-act-1989-for-a-child-arrangements-prohibited-steps-specific-issue-section-8-order-or-to-vary-or-discharge' class='govuk-link' rel='external' target='_blank'>Form C100 on GOV.UK</a>. Download the from and fill in your details. -welsh",
```
[the remark this answers is not in the document]
Comment (SasankAchyuta9) [reply src/main/steps/complete-your-application-paper-form/content.ts:19]: Yeah populated the fields

**Review by vivek-sekhar — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**